src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+const body = document.querySelector('body');
+const div = document.createElement('div');
+
+div.dataset.qa = 'notification';
+
+const firstPromise = new Promise((resolve, reject) => {
+  body.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(Error('First promise was rejected'));
+  }, 3000);
+});
+
+firstPromise
+  .then((result) => {
+    div.className = 'succes';
+    div.textContent = result;
+    body.append(div.cloneNode(true));
+  })
```

Comment:
create separate function `addNotification` for successful and rejected result (make it common by propagating additional param) and use it everywhere.